doc/source/reference/random/generator.rst

```diff
@@ -97,16 +97,16 @@ which dimension of the input array to use as the sequence. In the case of a
 two-dimensional array, ``axis=0`` will, in effect, rearrange the rows of the
 array, and  ``axis=1`` will rearrange the columns.  For example
 
-    >>> rng = np.random.default_rng()
+    >>> rng = np.random.default_rng(2345)
```

Comment:
Same comment here for seed and `# random`

Reply:
applies to this whole file

Reply:
Also done.